acoustid/scripts/backfill_meta_gid.py

```
@@ -37,20 +38,26 @@ def update_last_meta_id(fingerprint_db, last_meta_id):
+    for meta in fingerprint_db.execute(query).all():
+        meta_id = meta.id  # type: int
         if meta_id > last_meta_id:
             last_meta_id = meta_id
-        if meta["gid"]:
+        if meta.gid:
             continue
-        meta_gid = generate_meta_gid(dict(meta))
-        query = sql.select([tables.meta.c.id], tables.meta.c.gid == meta_gid)
-        new_meta_id = fingerprint_db.execute(query).scalar()
+
+        meta_dict = {col: getattr(meta, col) for col in meta._mapping.keys()}
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)